src/components/Heading/index.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../styles/variables.scss";
+
+.Heading-light {
+  font-weight: $font-weight-light;
+}
+
+.Heading-regular {
+  font-weight: $font-weight-regular;
+}
+
+.Heading-bold {
+  font-weight: $font-weight-bold;
+}
+
+.Heading-xsmall {
+  font-size: $font-size-xsmall;
+}
+
+.Heading-level1 {
+  font-size: 2em;
+}
+
+.Heading-level2 {
+  font-size: 1.5em;
+}
+
+.Heading-level3 {
+  font-size: 1.17em;
+}
+
+.Heading-level4 {
+  font-size: 1em;
+}
+
+.Heading-level5 {
+  font-size: .83em;
```

Comment:
You guys are using `em` at Oncostats? How do these differ from the `$font-size` variables?

Reply:
We still have some media queries, but I plan to remove them after some testing.

Reply:
I got what you were going for with `em` but how do these differ from the `$font-size` variables? They are also in `em`

Reply:
Yep, needs some cleaning up. The components will be using the variables for the sizes. The variables will be useful for possible scenarions (we are already using a font size variable to set the navbar bottom spacing)

Reply:
$font-sizes will be variables with `em` sizings, I forgot to refactor that bit and directly specified the `em` sizes on the Heading and Text components.